openapi/components/schemas/assessment-feedback-list.yaml

```diff
@@ -30,15 +30,34 @@ properties:
               example: dueWork
             commonAssessment:
               type: boolean
-            weight:
-              type: string
-              example: 1.00
+            weighted:
```

Comment:
Interesting: we're only returning whether an assessment is weighted, not its actual weight.

There might not be much point to returning an assessment's weighting until we provide an easy method for fetching assessments by term, though.

Reply:
thoughts?
return weight value, or leave as boolean?